src/main/java/exceptions/ExceptionAlreadyTaskMarkedDone.java

```diff
@@ -0,0 +1,4 @@
+package exceptions;
+
+public class ExceptionAlreadyTaskMarkedDone extends Exception {
```

Comment:
Since a class represents the blueprint of an instance/object - it would be better to name it accordingly (more noun-ish).  Try ending your exception class names with Exception. For example `TaskAlreadyMarkedException`. Maybe try refactoring the name similarly for your other exception classes as well?

https://nus-cs2113-ay2021s2.github.io/website/se-book-adapted/chapters/refactoring.html